packages/shell-chrome/views/_partials/data/attribute.edge

```diff
@@ -69,14 +71,11 @@
 
     <div
         class="flex flex-col"
-        x-show="!singleData.hasArrow
-            && !singleData.readOnly
-            && singleData.isOpened"
+        x-show="!singleData.hasArrow && !singleData.readOnly && (singleData.depth === 0 || singleData.isOpened)"
```

Comment:
Wrapping this in `<template x-if` instead of `x-show` will only have Alpine render this part on demand when the user presses to open it, going from `O(n^n)` to `O(n)` (right? Sounds impressive anyway)

Reply:
In this file we also have `$nextTick()` (twice) which I don't think is useful here. It's much more responsive without it and all seemed to work fine as well. If anything maybe wrap it in `requestAnimationFrame` instead?

Reply:
Shall I merge this and we can do another PR for these optimisations?

Really good spots Kevin 🙂

Reply:
Created a separate PR https://github.com/alpine-collective/alpinejs-devtools/pull/182

we can either merge into this one or merge this into master and then start doing the data optimisations

Reply:
Merged it

Let me know if you find any issues/other improvements otherwise will look to merge this one, ready for a 1.0 next week